run-iblockdevice-acceptance-tests

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
```

Comment:
Not going to spend much time reviewing this since it's going to be thrown out soon, I hope.